src/jobs/build-and-push-image.yml

```diff
@@ -233,19 +249,43 @@ steps:
         - run: |
             docker login -u $<<parameters.dockerhub-username>> -p $<<parameters.dockerhub-password>>
 
-  - build-image:
-      registry-id: <<parameters.registry-id>>
-      repo: <<parameters.repo>>
-      tag: <<parameters.tag>>
-      dockerfile: <<parameters.dockerfile>>
-      path: <<parameters.path>>
-      extra-build-args: <<parameters.extra-build-args>>
-      no-output-timeout: <<parameters.no-output-timeout>>
-      skip-when-tags-exist: <<parameters.skip-when-tags-exist>>
-      profile-name: <<parameters.profile-name>>
-      platform: <<parameters.platform>>
-      region: <<parameters.region>>
-      public-registry: <<parameters.public-registry>>
-      push-image: <<parameters.push-image>>
-  - store_artifacts:
-      path: test.txt
```

Comment:
i understand this was likely erroneously committed, hence i removed it in this PR.

Reply:
good catch!